src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
I appreciate trying to keep our options open for what we tell users & how we respond, but in this case, I agree that the user is probably not in the best position to appreciate the distinctions here (out of proc is arcane enough for us, and in theory we know something about it).  For now, let's go with simple as Cyrus and Paul suggests.